specification/storage/resource-manager/Microsoft.Storage/stable/2023-04-01/storage.json

```diff
@@ -0,0 +1,5774 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AzureStorageManagement",
```

Comment:
could you revert all the titles to ```StorageManagementClient```?

Reply:
Thanks for your comment, can you please specify the reason?

Reply:
JS SDK breaking is caused by changing the title to AzureStorageManagement. Does it make other language breaking changes if uses StorageManagementClient?

Reply:
I don't know how other SDK works, only SDK for go validation complains the title not same cross these json, so trying to use a standard one, just update all the title to "StorageManagementClient"